modular_skyrat/modules/sec_haul/code/guns/pepperball_gun.dm

```diff
@@ -1,74 +0,0 @@
-/obj/item/gun/ballistic/automatic/pistol/pepperball
-	name = "\improper Bolt Pepperball AHG"
-	desc = "An incredibly mediocre 'firearm' designed to fire soft pepper balls meant to easily subdue targets."
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/pepperball.dmi'
-	icon_state = "peppergun"
-	w_class = WEIGHT_CLASS_NORMAL
-	accepted_magazine_type = /obj/item/ammo_box/magazine/pepperball
-	can_suppress = FALSE
-	fire_sound = 'sound/effects/pop_expl.ogg'
-	rack_sound = 'sound/weapons/gun/pistol/rack.ogg'
-	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
-	bolt_drop_sound = 'sound/weapons/gun/pistol/slide_drop.ogg'
-	fire_sound_volume = 50
```

Comment:
This one is one that I disagree pretty heavily with the removal of. I think it was fine as it was, it had interesting mechanics and felt different from the rest.

Reply:
The pepperball manages to both be the strongest and most useless weapon in the codebase. The determining factor being "did they go to a fire locker before the fight" and the item that's easiest to completely invalidate the weapon with having no downsides at all and being so comically easy to access makes the unique mechanics of the weapon just frustrating. If someone is wearing a mask then your ranged weapon does literally nothing to them. If someone is not wearing a mask, then your ranged weapon has become the strongest weapon in the game. This is not intuitive in any way to people who don't know exactly what everything does and is a particularly unfun way to have something work.